packages/browser-sdk/src/toolbar/Toolbar.tsx

```diff
@@ -76,6 +76,10 @@ export default function Toolbar({
   const searchedFeatures =
     search === null ? features : features.filter((f) => f.key.includes(search));
 
+  const sortedFeatures = searchedFeatures.sort((a, b) =>
+    a.key.localeCompare(b.key),
+  );
```

Comment:
cool but `.sort()` unfortunately updates `searchedFeatures` so the extra variable/assignment just adds noise.

Reply:
right, went a tad too fast 🙈 fixed